app/controllers/health_card_controller.rb

```diff
@@ -0,0 +1,71 @@
+require 'health_cards'
+require 'open-uri'
+
+class HealthCardController < ApplicationController
+  class << self
+    @vaccine_groups = nil
+
+    def vaccine_groups
+      @vaccine_groups ||= load_vaccine_groups
+    end
+
+    def load_vaccine_groups
+      f = File.open(Rails.root.join('app', 'assets', 'iisstandards_vax2vg.xml'))
+      @vaccine_groups = Nokogiri::XML(f)
+    end
+  end
+
+  @private_key = nil
+  @issuer = nil
+
+  skip_before_action :verify_authenticity_token, only: [:create]
+
+  def index; end
+
+  def create
+    # respond_to do |format|
+    #  format.fhir_json do
+    fhir_params = FHIR.from_contents(request.raw_post)
```

Comment:
not sure if we should accept outside parameters directly, so maybe refactor this into a private method, and later we can add security measures to it.

Reply:
I'm not sure what you mean by this. We have to accept FHIR::Parameters in order to comply with the $Issue spec.